src/test/java/com/skiwi/githubhooksechatservice/mvc/beans/tasks/CommentsScanTaskTest.java

```diff
@@ -0,0 +1,35 @@
+import java.util.regex.*;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class CommentsScanTaskTest {
+	private static Pattern interestingComment = Pattern.compile("[\\*`_]{0,3}code[\\*`_]{0,3}\\s*[\\*`_]{0,3}review[\\*`_]{0,3}");
+	
+	private static boolean isInterestingComment(String comment) {
+		String commentText = comment.toLowerCase(Locale.ENGLISH);
+		return interestingComment.matcher(commentText).find(0);
+	}
+	
+	public static void test() throws Exception {
```

Comment:
It might additionally be interesting to not copy-paste the source of the tested method into the test-class because that somewhat defeats the point of examining the actual codebase...